go/database/mpt/io/live.go

```diff
@@ -72,6 +72,8 @@ func Export(directory string, out io.Writer) error {
 	}
 	defer db.Close()
 
+	ctx := catchInterrupt()
```

Comment:
Same argument as for the archive export. I think this is a too narrow scope to deal with interrupt handler.